pyccel/naming/languagenameclashchecker.py

```diff
@@ -19,6 +19,11 @@ class LanguageNameClashChecker(metaclass = Singleton):
     """
     keywords = None
 
+    def __init__(self): #pylint: disable=useless-parent-delegation
+    # This __init__ function is required so the ArgumentSingleton can
+    # always detect a signature
+        super().__init__()
```

Comment:
For ease of parsing I would recommend putting the comment inside the function
```suggestion
    def __init__(self): #pylint: disable=useless-parent-delegation
        # This __init__ function is required so the ArgumentSingleton can
        # always detect a signature
        super().__init__()
```

Reply:
[8cc3231](https://github.com/pyccel/pyccel-cuda/pull/60/commits/8cc32315d4c1a8020b2c3c1aaa0189d03ef82257)